__tests__/AutocompleteInput.spec.js

```diff
@@ -72,4 +72,50 @@ describe('<AutocompleteInput />', () => {
 
     expect(textInput.props).toEqual(expect.objectContaining(props));
   });
+
+  it('should render default <FlatList /> if no custom one is supplied', () => {
```

Comment:
Technically this test isn't necessary. It's redundant with the first test. But I wanted to add a test that tests that the default result list still works correctly and correctly passes `data` to the `<FlatList />` component.